srcpkgs/freecad/template

```diff
@@ -15,7 +15,8 @@ _inst_prefix=/usr/lib/${pkgname}
 configure_args="
  -DBUILD_QT5=OFF
  -DCMAKE_INSTALL_PREFIX=${_inst_prefix}
- -DCMAKE_INSTALL_LIBDIR=${_inst_prefix}/lib"
+ -DCMAKE_INSTALL_LIBDIR=${_inst_prefix}/lib
+ -DMEDFILE_INCLUDE_DIRS=/usr/include/med"
```

Comment:
should problably be ${XBPS_CROSS_BASE} so that it expands to a cross-compilation friendly path when cross building.